src/middlewares/transform.middleware.ts

```diff
@@ -0,0 +1,58 @@
+import { NextFunction, Request, Response } from 'express';
+import { dayjs } from '@utils/dayjs.util';
+
+export const transform = () => {
+  return (body: Object, req: Request, res: Response) => {
+    const timestamp = dayjs().utc().format();
+
+    if (Array.isArray(body)) {
+      let page = Number(req.query.page);
+      let limit = Number(req.query.limit);
+
+      if (page < 1) {
+        page = 1;
+      }
+
+      const MAX_LIMIT = 2;
+
+      if (limit > MAX_LIMIT) {
+        limit = MAX_LIMIT;
+      }
+
+      const startIndex = (page - 1) * limit;
+      const endIndex = page * limit;
+      const item = body.slice(startIndex, endIndex);
```

Comment:
The body array that you're getting will be limited from the service / repository level, it shouldn't be done here